src/api-binder/report.ts

```diff
@@ -166,5 +196,5 @@ export async function startReporting() {
 
 	// Start monitoring for changes that do not trigger deviceState events
 	// Example - device metrics
-	return recursivelyReport(constants.maxReportFrequency);
+	return recursivelyReport(constants.maxMetricsFrequency);
```

Comment:
We discussed "resetting the metrics report timer when a device-state change is reported" -- is that still the case? It looks like the recursive report will fire every maxMetricsFrequency (5min) regardless of whether a deviceState 'change' event has occurred / been reported.

Reply:
Thanks, IMO the device has a contract with the cloud that metrics will be reported at a defined maximum frequency, and the cloud enforces this contract. However, I will revise the code to make it easy to change this decision.